aiosmtpd/controller.py

```diff
@@ -72,16 +72,7 @@ def _stop(self):
         self.loop.stop()
         try:
             _all_tasks = asyncio.Task.all_tasks
-        except AttributeError:   # pragma: nocover
-            # I don't like using pragma:nocover just like everyone else, but at
-            # the time being this is the only way to achieve 100% coverage on
-            # 3.6 <= Python < 3.9. (One day *if* we drop support for 3.6, we
-            # can revisit and simplify this, because 3.7 implemented
-            # asyncio.all_tasks)
-            # That said, I'm planning to use coverage-conditional-plugin, but I
-            # have hit some problems that prevented me from using that plugin's
-            # 'conditional pragmas'. As soon as I can get that problem fixed,
-            # I'll replace the pragma:nocover.
+        except AttributeError:   # pragma: skipif_lt_py39
```

Comment:
Yay! We've killed the `nocover` and replaced it with a better "conditional" pragma.